pkg/dmesg/watcher.go

```diff
@@ -83,6 +91,8 @@ func (w *watcher) Close() {
 func watch(
 	ctx context.Context,
 	cmds [][]string,
+	cacheExpiration time.Duration,
+	cachePurgeInterval time.Duration,
 ) (<-chan LogLine, error) {
 	ch := make(chan LogLine, 1000)
```

Comment:
Should we need to extend the channel buffer size?

Reply:
done